packages/nuxt3/src/core/plugins/import-protection.ts

```diff
@@ -18,7 +18,7 @@ export const vueAppPatterns = (nuxt: Nuxt) => [
   [/(^|node_modules\/)@vue\/composition-api/],
   ...nuxt.options.modules.filter(m => typeof m === 'string').map((m: string) =>
     [new RegExp(`^${escapeRE(m)}$`), 'Importing directly from module entrypoints is not allowed.']),
-  ...['#static-assets', '#static', '#config', '#assets', '#storage', '#server-middleware']
+  ...['#static-assets', '#static', '#assets', '#storage', '#server-middleware']
```

Comment:
I think we can revert it back:
```suggestion
  ...['#static-assets', '#static', '#config', '#assets', '#storage', '#server-middleware']
```

Reply:
No, it will warn because of the virtual template we create internally.

Reply:
Can we use `#_config` within virtual template then? (i'm fine even if it requires an additional alias with nitro but not allow `#config` import in client. it is addictive and potentially breaking change in the future)